src/components/HeaderForm.js

```diff
@@ -540,7 +575,8 @@ HeaderForm.propTypes = {
   dataOrigin: PropTypes.string.isRequired,
   setColorSetting: PropTypes.func.isRequired,
   setDataOrigin: PropTypes.func.isRequired,
-  setFetchParams: PropTypes.func.isRequired
+  setCurrentViewTarget: PropTypes.func.isRequired,
+  viewTarget: PropTypes.any, // Header Form State, may be null if no params in URL. see Types.ts
```

Comment:
Since there's already code in App to pick a default ViewTarget if one isn't in the URL parameters, maybe this should be a required prop, so we don't need to look in the config inside this component as well?

It also might make sense to call it `defaultViewTarget` instead of `viewTarget`, because it's yet a third ViewTarget floating around, in addition to the "current" and "next" ones, and they need to be kept straight.